File: src/app/Emprestimo.php
<?php

namespace App;
use App\Usuario;
use App\Livro;

class Emprestimo
{
    private  int $id;
    private Usuario $usuario;
    private Livro $livro;

    public function __construct(int $id, Usuario $usuario, Livro $livro)
    {
        $this->id = $id;
        $this->usuario = $usuario;
        $this->livro = $livro;
    }

    public function getId(): int
    {
        return $this->id;
    }
    
    public function getUsuario(): Usuario
    {
        return $this->usuario;
    }

    public function getLivro(): Livro
    {
        return $this->livro;
    }

}

File: 1-modelagem-codigo-python/app/emprestimo.py
from app.usuario import Usuario
from app.livro import Livro

class Emprestimo:
    def __init__(self, id: int, usuario: Usuario, livro: Livro):
        self.__id = id
        self.__usuario = usuario
        self.__livro = livro

    def get_id(self) -> int:
        return self.__id

    def get_usuario(self) -> Usuario:
        return self.__usuario

    def get_livro(self) -> Livro:
        return self.__livro